lib/foodsoft_mail_receiver.rb

```diff
@@ -0,0 +1,34 @@
+require 'mail'
+require 'mini-smtp-server'
```

Comment:
I had trouble to `mini-smtp-server` as explained in https://github.com/foodcoops/sharedlists/issues/3#issuecomment-328119403.